### ISSUE TITLE
Does this support yolov5 4.0 release?

### ISSUE THREAD

**Issue by KevinZhanggg** [positive666/Deepstream_Project#1]

**Comment by positive666** [comment]
Yes support, 4.0 release should require torch 1.7. The conversion model provides two project conversion methods under trans_project. Recently, I will supplement this project when I have time. You can try to convert

**Comment by KevinZhanggg** [comment]
@positive666 我用tensorrtx 那个转换出来的的engine 可以推理出来，输出的结果也很正常。 但是放在deepstream 里之后推理就有很多boxes，很多误检。 你有遇到这个情况吗？

**Comment by positive666** [comment]
之前没遇到过，但你说TensorRt没问题前提下，这里可能有两个具体原因？是否你的deepstream下的 解析cpp代码中NMS有问题 ；或者是阈值问题 你可以调一下看看，我在第一次作V3的模型转换时候遇到过，类似问题，当时是Nms的阈值造成的

**Comment by KevinZhanggg** [comment]
@positive666 我感觉是NMS的问题，但没先找出来具体那里需要修改，阈值大小我修改过，还是一样。冒昧的问下，可以在给个微信或者qq吗，方便的话，想讨论下。谢谢🙏

**Comment by positive666** [comment]
没问题 互相学习 我最近在别的开发项目中 还没来得及更新 wechat:cr286040359




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "KevinZhanggg"<notifications@github.com&gt;; 
发送时间: 2021年2月19日(星期五) 中午12:16
收件人: "positive666/Deepstream_Project"<Deepstream_Project@noreply.github.com&gt;; 
抄送: "CR"<286040359@qq.com&gt;; "Mention"<mention@noreply.github.com&gt;; 
主题: Re: [positive666/Deepstream_Project] Does this support yolov5 4.0 release? (#1)





 
@positive666 我感觉是NMS的问题，但没先找出来具体那里需要修改，阈值大小我修改过，还是一样。冒昧的问下，可以在给个微信或者qq吗，方便的话，想讨论下。谢谢🙏

**Comment by positive666** [comment]
这个问题 就是tensorrt生成引擎文件不能在当前目录下生成 要使用非根目录
